.github/workflows/ci.yml

```diff
@@ -20,17 +20,13 @@ jobs:
           toolchain: stable
       - name: Chown cargo registry because of caching
         run: sudo chown -R runner ~/.cargo/registry
-      - name: Cache cargo registry
-        uses: actions/cache@v1
+      - name: Cache cargo registry and target
+        uses: actions/cache@v2
         with:
-          path: ~/.cargo/registry
-          key: registry-${{ hashFiles('**/Cargo.lock') }}
-      - name: Cache target
-        id: cache-target
-        uses: actions/cache@v1
-        with:
-          path: target
-          key: check-target-${{ steps.toolchain.outputs.rustc_hash }}-${{ hashFiles('**/Cargo.lock') }}
+          path: |
+            ~/.cargo/registry
+            target
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
```

Comment:
I think, we should keep the compiler version (`steps.toolchain.outputs.rustc_hash`) in the cache key.

Reply:
Thanks, I replaced `runner.os` (this was referenced in GH examples) with `steps.toolchain.outputs.rustc_hash`, which was used previously.